### ISSUE TITLE
key errors when using dictionary in the discum library on this program. 

### ISSUE THREAD

**Issue by dave-es** [vivinano/MudaeAutoBot#244]
![image](https://github.com/vivinano/MudaeAutoBot/assets/117441327/9493918b-e0e7-4044-aa36-ceb5c696dd30)
this error message pops up whenever trying to run the code

**Comment by JuliaYuki** [comment]
i was having the same error and I find this topic on Discum, this solution solved the error for me.
[https://github.com/Merubokkusu/Discord-S.C.U.M/issues/432#issuecomment-1360419048](url)

**Comment by rodrigues159** [comment]
@vivinano I will test with another branch of discum... but I found it very strange that it stopped working from one hour to another without having done any update in the code... 
See I sent you a message on Patreon, if you can answer thank you for not polluting the github :)

**Comment by JuliaYuki** [comment]
now it's back to giving the same error, now that method I found doesn't work anymore. Strange that it started all of a sudden

**Comment by JuliaYuki** [comment]
I tried uninstalling and installing again in the same way, but unfortunately it didn't work... Funny that if I use another token from another account, it works perfectly, but with the account I want to use, it started giving this error.

**Comment by rodrigues159** [comment]
@JuliaYuki yes yes, I tested it with another account too and it is working perfectly just the one I was using normally stopped working from one hour to another

**Comment by JuliaYuki** [comment]
@rodrigues159 If you manage to come up with a solution, please let me know.

**Comment by rodrigues159** [comment]
> @rodrigues159 If you manage to come up with a solution, please let me know.

ok, I am together with vivinano via Discord trying to identify... but my second account that was working has stopped working now too

**Comment by JuliaYuki** [comment]
> > @rodrigues159 If you manage to come up with a solution, please let me know.
> 
> ok, I am together with vivinano via Discord trying to identify... but my second account that was working has stopped working now too

What a strange error. I hope you guys manage to find a solution. I'm asking on Discum, looking for someone to help. Since I'm still very noob with python language

**Comment by JuliaYuki** [comment]
@rodrigues159 Bruh, I did the simplest thing now and it solved it... I still don't know the reason for giving this error but here we go:
I just downloaded MudaeAutoBot.rar again, configured my settings_mudae and executed again. It's been working perfectly so far.
